### ISSUE TITLE
Some ONIX accessibility codes missing

### ISSUE THREAD

**Issue by LaurenceZaysser** [w3c/a11y-discov-vocab#154]
All ONIX 196 code list accessibility codes do not find an equivalent in EPUB/Schema.org yet. In the spec, this has been properly indicated with an hypen (in the EPUB/Schema column). Though, it has not been done systematically. Some ONIX accessibility codes remain missing in the spec. But we (readers) don't know if it's a volountary action (and why?!), or if it's only an oversight. It would be much valuable to keep trace of the reasons why some ONIX values has been discarded from the crosswalk. This would avoid everybody (authors, and readers of the spec) to reanalyse these codes... Thanks ! ;-)

- 196-01 LIA Compliance Scheme [not an international label, just an Italian one?]
- 196-09 Inaccessible, or known limited accessibility [lack in Schema.org?]
- 196-10 No reading system accessibility options actively disabled [former accessControl value now deprecated?]
- 196-13 Single logical reading order [lack in Schema.org?]
- 196-18 Accessible chemistry content as ChemML [lack in Schema.org? Who knows, ChemML might be supported one day!]
- 196-25 Use of color is not sole means of conveying information [lack in Schema.org?]
- 196-29 Next / Previous structural navigation [reading system, not EPUB?]
- 196-31 Accessible controls provided (timing controls) [lack in Schema.org? - school/academic domain]
- 196-36 Appearance of all textual content can be modified [reading system, not EPUB?]
- 196-38 Unusual words or abbreviations explained [lack in Schema.org? - school/academic domain]
- 196-39 Supplementary material to an audiobook is accessible [usecase unknown?]
- 196-40 Link purposes clear [lack in Schema.org?]
- 196-41 Page list navigation [lack in Schema.org?]
- 196-51 All non-decorative content supports reading via pre-recorded audio [part of ONIX 196-52?]
- 196-52 All non-decorative content supports reading without sight [lack in Schema.org? But it could be both accessModeSufficient=textual|auditory for visually impaired! I am afraid the accessModeSufficient is an hybrid (handicap/alternative) not really focused on each handicap]
- 196-91 Latest accessibility assessment date [lack in Schema.org?]
- 196-92 Accessibility addendum [lack in Schema.org?]
- 196-95 Trusted intermediary’s web page for detailed accessibility information [lack in Schema.org?]
- 196-97 Compatibility tested [lack in Schema.org?]
- 196-98 Trusted Intermediary contact [lack in Schema.org?]
- 196-99 Publisher contact for further accessibility information [lack in Schema.org?]
